src/graphnet/models/components/embedding.py

```diff
@@ -93,6 +92,7 @@ def __init__(
                 f"{n_features} features."
             )
         elif n_features >= 6:
+            self.aux_emb = nn.Embedding(2, seq_length // 2)
```

Comment:
Looks like your recent commit added this line. Was that intentional?

Reply:
Ah no, sorry that was for another PR.